app/grandchallenge/archives/views.py

```diff
@@ -558,6 +564,115 @@ def get_serializer_class(self):
             return ArchiveItemSerializer
 
 
+class ArchiveItemJobListView(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    JobsList,
+):
+    model = Job
```

Comment:
This can go, right? It's already set on `JobsList`

Reply:
You can disregard this comment given the other comments about inheriting directly from `PaginatedListView`

Reply:
Opted to directly inherit PaginatedListView